### PR TITLE
[CARBONDATA-4010] Doc changes for long strings.

### PR DESCRIPTION
**Pull request by nihal0107**
### Why is this PR needed?
 Added documentation change for the handling of long strings(length greater than 32000) as bad record and  set/unset of longStringColumns.
 
 ### What changes were proposed in this PR?
 Added documentation change for the handling of long strings(length greater than 32000) as bad record and  set/unset of longStringColumns.
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2471/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4214/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2548/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4296/

**Comment by nihal0107** [comment]
restest this please.

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4300/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2552/

File: docs/ddl-of-carbondata.md [hunk 1]
```diff
@@ -426,7 +426,8 @@ CarbonData DDL statements are documented here,which includes:
    - ##### String longer than 32000 characters
 
      In common scenarios, the length of string is less than 32000,
-     so carbondata stores the length of content using Short to reduce memory and space consumption.
+     so carbondata stores the length of content using Short to reduce memory and space consumption, and it handles the strings
+     which length greater than 32000 as bad record. Refer [bad record handling](https://github.com/apache/carbondata/blob/master/docs/dml-of-carbondata.md#bad-records-handling) section for better understanding.
```
Comment (Indhumathi27) [inline docs/ddl-of-carbondata.md:430]:
```suggestion
 and it handles strings which have length greater than 32000 as a bad record.

```

Comment (nihal0107) [reply docs/ddl-of-carbondata.md]: done

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2566/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4316/

**Comment by Indhumathi27** [comment]
retest  this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4320/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2570/

File: docs/ddl-of-carbondata.md [hunk 2]
```diff
@@ -812,7 +813,19 @@ Users can specify which columns to include and exclude for local dictionary gene
...
+       ```
+       ALTER TABLE tablename SET TBLPROPERTIES('LONG_STRING_COLUMNS'='column1')
+       ```
+       **NOTE:** Only string columns can be set to long string columns. Cannot set sort columns to long string columns.
 
+       Example to UNSET Long String Columns:
+       ```
+       ALTER TABLE tablename UNSET TBLPROPERTIES('LONG_STRING_COLUMNS')
+       ```
+       **NOTE:** On unset long string columns are set to their original datatypes.
```
Comment (Indhumathi27) [inline docs/ddl-of-carbondata.md:827]:
```suggestion
       **NOTE:** On unset, long string columns are set to their original datatypes.
```

Comment (nihal0107) [reply docs/ddl-of-carbondata.md]: done

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4332/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2582/

**Comment by Indhumathi27** [comment]
LGTM

**Comment by Indhumathi27** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2615/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4365/

**Comment by nihal0107** [comment]
retest this please.

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/2620/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/4370/

**Pull request closed without merging**